pkg/capr/planner/certificaterotation_test.go

```diff
@@ -221,7 +221,7 @@ func Test_rotateCertificatesPlan(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
+			//t.Parallel()
```

Comment:
If this is not necessary, let's remove this code instead of commenting out.

Reply:
@snasovich updated the PR